src/helpers.js

```diff
@@ -240,20 +231,23 @@ export async function processESLintMessages(response, textEditor, showRule, work
         range = generateRange(textEditor, msgLine, msgCol)
       }
       ret = {
-        filePath,
-        type: severity === 1 ? 'Warning' : 'Error',
-        range
+        severity: severity === 1 ? 'warning' : 'error',
+        location: {
+          file: filePath,
+          position: range
+        }
       }
 
-      if (showRule) {
-        const elName = ruleId ? 'a' : 'span'
-        const href = ruleId ? ` href="${ruleURI(ruleId).url}"` : ''
-        ret.html = `${escapeHTML(message)} (<${elName}${href}>${ruleId || 'Fatal'}</${elName}>)`
-      } else {
-        ret.text = message
-      }
+      // Fallback to ruleId for triggering a Google search
+      // Useful for custom rules, that are not matched by `ruleURI`
+      const url = ruleId ? ruleURI(ruleId).url : ruleId
```

Comment:
Although this is currently the behavior of `linter`, it's not documented and is slated for removal. If providers are giving a `url` field, it _should_ be a full URL.

I would much prefer the current behavior which defaults to [this document](https://github.com/jfmengels/eslint-rule-documentation/blob/master/contributing.md), leading people on how to add the documentation. If they just want to search they can copy/paste from the panel.

If @IanVS want's though I would be okay with checking the result from `ruleURI(ruleID)`, and if `found` is `false` then we simply don't specify a `url` attribute.